hammock-asynchttpclient/src/test/scala/hammock/asynchttpclient/AsyncHttpClientInterpreterSpec.scala

```diff
@@ -3,15 +3,16 @@ package asynchttpclient
 
 import cats.implicits._
 import cats.effect._
-import org.scalatest._
+import org.scalatest.Matchers
+import org.scalatest.wordspec.AnyWordSpec
 import org.asynchttpclient._
 import io.netty.handler.codec.http.{DefaultHttpHeaders, HttpHeaders}
 import io.netty.handler.codec.http.cookie.Cookie
-import org.scalatest.mockito._
+import org.scalatestplus.mockito._
 import AsyncHttpClientInterpreter._
 import scala.collection.JavaConverters._
 
-class AsyncHttpClientInterpreterSpec extends WordSpec with Matchers with MockitoSugar {
+class AsyncHttpClientInterpreterSpec extends AnyWordSpec with Matchers with MockitoSugar {
```

Comment:
Out of curiosity: do you use `MockitoSugar`?

Reply:
Yeah, I use mockito to mock the integration's http driver... I'd love to find a better approach though :)

Reply:
I am not a big fan of mocking tools but I've heard about http://scalamock.org/
we can take a look after this PR if you want :)

Reply:
Oh thanks! Yes please, go ahead :)